vue/src/components/siteList/SiteList.vue

```diff
@@ -283,12 +283,12 @@ const startDownload = async (data: DownloadSettings) => {
   const id = imageDownloadingId.value;
   imageDownloadDialog.value = false;
   if (id) {
-  await ApiService.getObservationImages(id, data);
-  // Notify the ModelRunList that downloading is happening
-  if (imageDownloadingModelRunId.value) {
-    emit('image-download', imageDownloadingModelRunId.value);
-  }
-  state.downloadingCheck += 1;
+    await ApiService.getObservationImages(id, data);
+    // Notify the ModelRunList that downloading is happening
+    if (imageDownloadingModelRunId.value) {
+      emit('image-download', imageDownloadingModelRunId.value);
+    }
+    refreshModelRunDownloadingStatuses();
```

Comment:
I assume this was autoformatted. I'll file an issue to investigate why the linting step didn't flag this the first time around

Reply:
I think the linting step isn't performing as intended when I run `npm run lint:fix`, but I haven't looked into it yet.